src/main/webapp/script.js

```diff
@@ -12,3 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+function getGeolocation() {
+  if (navigator.geolocation) {
+    navigator.geolocation.getCurrentPosition(displayLocation, displayError);
+  }
+  else {
+    document.getElementById("location").innerHTML = "Your browser doesn't support geolocation - please enter zipcode";
```

Comment:
Perhaps raise this as an alert in the browser (a try/catch should do, I believe)? Filling an html element with a warning like this may not be the best user experience.

Reply:
Ah that's a great idea. I logged the issue to the console and created an alert that pops up on the top of the page. Let me know if this is better!